com/yadro/NetworkAdapter/Manager.interface.yaml

```diff
@@ -0,0 +1,6 @@
+description: >
+    Control Hardware Manager daemon
```

Comment:
I'm now wonder if it was good idea to add that com.yadro.Storage.Manager interface as a separate category... Probably it should be com.yadro.HWManager.Manager to not duplicate same interfaces.

Reply:
Yes, I was thought too about it. But I don't desire to change the previous implementation `Storage` in the context 'Add support Network Adapter manager'.
So, we should decide, does it is required to refine Storage implementation. Please, note, that changes will impact on the IPMI implementation, the part where we parse incoming storage information from the host.

Reply:
Right, this will affect two places - hwmanager and ipmi plug in. But you modify both anyway.

Reply:
I guess that will separate changes and patches anyway.

Reply:
ok

Reply:
I like the idea about share the same interface. And now is a good time to do it.
If you don't want to mix these changes you could make it as separate PRs.
Although I see nothing wrong to combine these changes into one PR but by the different commits.

So, this PR looks wrong as it increases the code duplication.

Reply:
Fixed